docs/dyn/compute_alpha.securityPolicies.html

```diff
@@ -194,7 +197,7 @@ <h3>Method Details</h3>
 </div>
 
 <div class="method">
-    <code class="details" id="insert">insert(project, body, requestId=None)</code>
+    <code class="details" id="insert">addRule(project, body, requestId=None)</code>
   <pre>Creates a new policy in the specified project using the data included in the request.
```

Comment:
ISTM that we need entries for both [`insert`](https://cloud.google.com/compute/docs/reference/rest/beta/securityPolicies/insert) and [`addRule`](https://cloud.google.com/compute/docs/reference/rest/beta/securityPolicies/addRule) (both methods are supported in the [API reference].  The entry for `addRule` should be at the top (since we are sorting alphabetically), and should have an `id` of `addRule`.